### DISCUSSION TITLE
ETH does not show on my Zksync Era testnet metamask

### DISCUSSION THREAD

**Discussion by guha-rahul** [zkSync-Community-Hub/zksync-developers#153] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement.

I am trying to bridge my goerli eth to zksync era testnet

### What is the specific issue or error you're encountering?

Tho the transaction seems to be completing , i am not able to see the eth on my metamask wallet and i am on the correct network. And i am attaching 3 pictures supporting it.
![zksync](https://github.com/zkSync-Community-Hub/zkync-developers/assets/52607971/1fb51c6d-9ede-4bba-9297-ee1a9da0245a)
![zksync1](https://github.com/zkSync-Community-Hub/zkync-developers/assets/52607971/eafe0169-6960-4806-a7b6-2af4a54cf247)
![zksync2](https://github.com/zkSync-Community-Hub/zkync-developers/assets/52607971/6e0c1e30-1de6-4e65-8cc5-dcc19af215fb)

### Can you share the error messages or logs you're receiving, if any?

.

### Have you made any recent changes to the contract before encountering this issue?

.

### Are there any external libraries or contracts that your contract interacts with?

.

### Can you provide the relevant portions of your contract code where the issue is occurring?

.

### Have you tried to isolate the problem, and if so, what were the results?

.

### What steps have you already taken to try to resolve the issue?

.

**Comment by MexicanAce** [comment]
Hi @guha-rahul 👋 

That RPC Url looks incorrect for testnet, can you try again with the following connection information: https://era.zksync.io/docs/dev/building-on-zksync/interacting.html#connecting-to-zksync-era-on-metamask

**Comment by guha-rahul** [comment]
![image](https://github.com/zkSync-Community-Hub/zkync-developers/assets/52607971/19902d72-f6ec-4e0d-95a2-bc33c842a7da)
![image](https://github.com/zkSync-Community-Hub/zkync-developers/assets/52607971/826832f7-fd66-4c3c-8ded-38c421beca6c)
The result stays the same.

**Comment by guha-rahul** [comment, marked as answer]
SOLUTION FOUND:- Metamask wa showing orange balance which means it was not connected to the blockchain hence not updating. I just used a new one
